packages/components/src/playground/button.tsx

```diff
@@ -1,33 +1,20 @@
 import React from "react";
-import styled from "styled-components";
+import tw from "twin.macro";
 
 // Temporary props for button to test out css options for variants
 type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
   variant?: "primary" | "secondary";
 };
 
-const ButtonComponent = styled.button<ButtonProps>`
-  border: none;
-  border-radius: 0.25rem;
-  color: white;
-  font-weight: bold;
-  padding: 0.5rem 1rem;
+const ButtonComponent = tw.button`
+  font-bold py-2 px-4 rounded text-white border-none
```

Comment:
Is stylelint already testing these? The way I checked was to run `npm run lint:styles -- -f verbose` https://stylelint.io/user-guide/usage/cli#example-i---print-on-success

Reply:
yep, I see them in there! thanks for checking 👍